Koans/AboutLambdas.cs

```diff
@@ -123,5 +124,25 @@ public void BlockNotNeededOnSingleStatementLambdas()
 			//When you have only one statement, the curly brackets are not needed. What other two things are also missing?
 			Assert.Equal(FILL_ME_IN, anonymous);
 		}
+
+		[Step(8)]
+		public void HowToWhereWorks()
+		{
+			var numbers = new[] {1, 2, 3, 4};
+			var result = numbers.Where(x => x > 2).ToArray();
+			
+			//What walues will should be in array?
+			Assert.Equal(FILL_ME_IN, result);
+		}
+		
+		[Step(9)]
+		public void HowToSelectWorks()
```

Comment:
It looks like these two tests should be moved to AboutLinq. Also can you give them a name that describes what is happening a bit better? Check the other tests in about linq.

Reply:
changed. Right now the functions name will be good? :)